README.md

```diff
@@ -44,3 +54,33 @@ By default, stats are logged every 5 seconds, you can adjust that using
 ``STATSD_LOG_EVERY`` which is the number of seconds between logging operations:
 
     STATSD_LOG_EVERY = 5
+
+Set the desired prefix:
+
+    STATSD_PREFIX = 'scrapy'
+
+You can specify which prefixes you want logged if you don't want all scrapy
+stats. The default is an empty list which indicates that all stats should be
+logged. You can, for example, log only downloader and robotstxt exception stats
+by setting ``STATSD_LOG_ONLY`` to ``['downloader', 'robotstxt.exception_count']``.
+
+    STATSD_LOG_ONLY = []
+
+You can also specify prefixes to ignore the same way using ``STATSD_IGNORE``:
+
+    STATSD_IGNORE = []
+
+
+## Tags
+
+Certain platforms such as datadog and influxdb offer tagging options.
+
+To enable tagging set ``STATSD_TAGGING`` to ``True``, it is disabled by default:
+
+    STATSD_TAGGING = False
+
+Then, you can set custom tags using ``STATSD_TAGS``:
+
+    STATSD_TAGS = {
+        'spider_name': True
```

Comment:
This is wip, I'm trying to figure out a better way of allowing custom tags.
I'm thinking either continue with some defaults (add project name / version / date tags) and/or allow a method/attribute on the spider act on these

Reply:
Ah, didn't see this :sweat_smile:

Reply:
I'm :heavy_plus_sign: for attributes.  I have projects where we may have like a `provider` argument or `base_url` or whatever, which would be helpful to tag on.

Reply:
Added attribute 'statsd_tags' that should be a dictionary with additional tags. I think this is a good starting point and we can extend later.